WORKSPACE

```diff
@@ -247,9 +248,10 @@ llvm_toolchain(
         "linux-aarch64": "@linux_sysroot-aarch64//:sysroot",
     },
     urls = {
-        "darwin-aarch64": ["https://github.com/MaterializeInc/toolchains/releases/download/clang-{0}/darwin_aarch64.tar.zst".format(LLVM_VERSION)],
-        "linux-aarch64": ["https://github.com/MaterializeInc/toolchains/releases/download/clang-{0}-3/linux_aarch64.tar.zst".format(LLVM_VERSION)],
-        "linux-x86_64": ["https://github.com/MaterializeInc/toolchains/releases/download/clang-{0}-3/linux_x86_64.tar.zst".format(LLVM_VERSION)],
+        "darwin-aarch64": ["https://github.com/MaterializeInc/toolchains/releases/download/clang-{0}-4/darwin_aarch64.tar.zst".format(LLVM_VERSION)],
```

Comment:
I'd probably put the suffix close to line 218's, LLVM_VERSION, something like `LLVM_VERSION_SUFFIX`?

Reply:
Good call! Added